README.md

```diff
@@ -55,18 +62,18 @@ yarn
 
 ## Testing
 
-The tests can be run with [foundry](https://github.com/foundry-rs/foundry).
+The tests can be run with [Foundry](https://github.com/foundry-rs/foundry).
 
 For the `RedBlackBinaryTree`, you can run the tests with hardhat with `yarn test`.
 
 ## Creating issues and PRs
 
 Guidelines for creating issues and PRs:
 
-- Issues must be created and labelled with relevant labels (type of issues, high/medium/low priority, etc.).
+- Issues must be created and labeled with relevant labels (type of issues, high/medium/low priority, etc.).
 - Nothing should be pushed directly to the `main` branch.
-- Pull requests must be created before and branch names must follow this pattern: `feat/<feature-name>`, `test/<test-name>` or `fix/<fix-name>`. `docs`, `ci` can also be used. The goal is to have clear branches names and make easier their management.
-- PRs must be labelled with the relevant labels.
+- Pull requests must be created before and branch names must follow this pattern: `feat/<feature-name>`, `test/<test-name>` or `fix/<fix-name>`. `docs`, `ci` can also be used. The goal is to have clear branches' names and make easier their management.
+- PRs must be labeled with the relevant labels.
 - Issues must be linked to PRs so that once the PR is merged related issues are closed at the same time.
 - Reviewers must be added to the PR.
 - For commits, we use the bitmoji VS Code extension 🙃
```

Comment:
Opening a debate here: is it useful to have this in the README. Considering the number of OS contributors that we have, I feel that these precisions cause more harm than good.

Reply:
I think we can just keep that we should use conventional commits.

Reply:
I was talking about the whole "issues and PRs" subsection

Reply:
I think we can make it just shorter and simpler to say that we use conventional commits as said by Merlin.